api/db/database-builder/factory/build-session.js

```diff
@@ -26,12 +27,16 @@ const buildSession = function ({
   juryCommentedAt = null,
   supervisorPassword = 'PIX12',
   version = 2,
+  createdBy = null,
 } = {}) {
   if (_.isUndefined(certificationCenterId)) {
     const builtCertificationCenter = buildCertificationCenter();
     certificationCenter = builtCertificationCenter.name;
     certificationCenterId = builtCertificationCenter.id;
   }
+  if (_.isUndefined(createdBy)) {
```

Comment:
Je le vois passer de temps à autre dans une PR mais je n'arrive pas à saisir l'intérêt d'utiliser lodash dans ce cas de figure.

```suggestion
  if (createdBy === undefined) {
```

Reply:
Ca sert effectivement à pas grand chose



```typescript
/**
 * Checks if `value` is `undefined`.
 *
 * @since 0.1.0
 * @category Lang
 * @param {*} value The value to check.
 * @returns {boolean} Returns `true` if `value` is `undefined`, else `false`.
 * @example
 *
 * isUndefined(void 0)
 * // => true
 *
 * isUndefined(null)
 * // => false
 */
function isUndefined(value) {
    return value === undefined;
}

export default isUndefined;
```

A noter qu'il n'y a pas eu d'autre implementations dans lodash, par contre dans underscore, on a eu:

en [0.4.6](https://github.com/jashkenas/underscore/blob/0.4.6/underscore.js) 
```javascript
return typeof obj == 'undefined';
``` 
puis [ensuite](https://github.com/jashkenas/underscore/blob/master/modules/isUndefined.js)
```javascript
return obj === void 0;
```